.github/actions/nm-build-vllm-whl/action.yml

```diff
@@ -24,11 +30,19 @@ runs:
       source $(pyenv root)/versions/${{ inputs.python }}/envs/${VENV}/bin/activate
       SUCCESS=0
       pip3 wheel --no-deps -w dist . || SUCCESS=$?
-      echo "status=${SUCCESS}" >> "$GITHUB_OUTPUT"
       ls -alh dist/
       BASE=$(./.github/scripts/convert-version ${{ inputs.python }})
       WHL_FILEPATH=$(find dist -iname "*nm_vllm*${BASE}*.whl")
       WHL=$(basename ${WHL_FILEPATH})
       echo "whl=${WHL}" >> "$GITHUB_OUTPUT"
+      # generate .tar.gz
+      $SUCCESS || python3 setup.py sdist bdist_wheel || SUCCESS=$?
```

Comment:
does this take a lot of time since it might be rebuilding the package again?

Reply:
Good catch, sorry should only build the source package not the wheel. Updated code, thanks!